src/shared/websocket/pushmessage.h

```diff
@@ -18,26 +17,13 @@ class PushMessage final : public QObject {
 
   bool executeAction();
 
-  enum MessageType {
```

Comment:
With this change we need to update the description of the PushMessageReceived event metric. To make it generic. https://github.com/mozilla-mobile/mozilla-vpn-client/blob/c06ab8e1ce2bc8af81c85032bbeec59ffd776f07/glean/metrics.yaml#L1547